### PR TITLE
Python implementation of run script

### PR DESCRIPTION
**Pull request by jishi9**

### CODE REVIEW

File: run.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+            p.kill()
+        except:
+            pass
+
+
+def main():
+    server_args = []
+
+    run_command(['pip', 'install', '-e', '.'])
+    run_command(['./example_project/manage.py', 'migrate', '--noinput'])
```
Comment (Spycho) [inline run.py]: We'll likely need to put the python command in front of manage.py here, as on windows it doesn't know how to just execute the file. I'll test when I get a chance.

File: run.py [hunk 2]
```diff
@@ -40,12 +40,12 @@ def main():
     server_args = []
 
     run_command(['pip', 'install', '-e', '.'])
-    run_command(['./example_project/manage.py', 'migrate', '--noinput'])
-    run_command(['./example_project/manage.py', 'collectstatic', '--noinput'])
+    run_command(['python', './example_project/manage.py', 'migrate', '--noinput'])
+    run_command(['python', './example_project/manage.py', 'collectstatic', '--noinput'])
```
Comment (CelineBoudier) [inline run.py]: can't we use os.path.join there instead of  / ?

File: run.py (hunk 1 above)
Comment (jishi9) [inline run.py]: fixed

File: run.py (hunk 2 above)
Comment (jishi9) [inline run.py]: done

**Comment by CelineBoudier** [comment]
mayne squash the commits? Thanks :)

**Pull request merged**